### DISCUSSION TITLE
Potential Trojan

### DISCUSSION THREAD

**Discussion by tonpap** [nklayman/vue-cli-plugin-electron-builder#1816] [Q&A] [answered]
Hallo everyone,

I am using "vue-cli-plugin-electron-builder": "~2.1.1" "electron": "^12.0.0", and when I try install my app I get

 Trojan:Win32/Wacatac.B!ml
Detected by Microsoft Defender Antivirus

I saw that there is a problem with electron-builder and this is actually a false positive... https://github.com/electron-userland/electron-builder/issues/4878 
But why I cannot find release 2.1.1 here but only in https://www.npmjs.com/package/vue-cli-plugin-electron-builder/
Which is the version of electron-builder used in 2.1.1?

Thank you in advance

**Comment by MatthijsBurgh** [comment, marked as answer]
See https://github.com/nklayman/vue-cli-plugin-electron-builder/blob/7d0e71fbcfc44400ba2407349514c2308ed55913/package.json#L34

**Comment by tonpap** [comment]
Thanks a lot! 

Any suggestions on how to overcome the trojan issue?
Since using the same version of electron-builder for quite a while I get that trojan sporadically...
as also mentioned here https://github.com/electron-userland/electron-builder/issues/6474#issuecomment-1184670714

Thanks again!

**Reply by MatthijsBurgh** [reply]
This is an issue of electron-builder. I don't have the knowledge of that.

**Reply by tonpap** [reply]
Ok! thank you!
